controllers/controllers.js

```diff
@@ -52,3 +53,24 @@ exports.getAllArticles = (req, res, next) => {
     res.status(200).send({ articles: articles });
   });
 };
+
+exports.postComments = (req, res, next) => {
+  const { username: name } = req.body;
+  const { body: body } = req.body;
+  const { article_id: id } = req.params;
+  //// add way to check if ^^^ these exits probs ref array and includes
```

Comment:
To action this, you'll need to make the request for the specific article and see if that succeeds or fails. If it succeeds but we're still getting an empty array when we make the request for comments, we'll know that the array is empty because the article exists but there are no comments associated - rather than the article just not existing (which would also return an empty array if we hadn't already done the check described above)